src/ucp/core/ucp_worker.c

```diff
@@ -2159,6 +2160,11 @@ ucs_status_t ucp_worker_get_ep_config(ucp_worker_h worker,
                                         UCP_FEATURE_AM, UCP_OP_ID_AM_SEND,
                                         UCP_PROTO_FLAG_AM_SHORT, key->am_lane,
                                         &ep_config->am_u.max_eager_short);
+
+        ucp_worker_ep_config_short_init(worker, ep_config, ep_cfg_index,
```

Comment:
Sure.
It's related to this PR, but a bit indirectly.
While testing this little feature I discovered that existing `max_short` limits are correctly adjusted based on current RNDV value. Namely the following short limits were properly adjusted:
```
tag.max_eager_short
tag.offload.max_eager_short
am_u.max_eager_short
```
The adjustment basically means that short limit can't exceed the RNDV threshold. My understanding is that this adjustment is needed because we want switch to RNDV protocol even if configured value of RNDV threshold is below some "short limits".
However the very similar short threshold `am_u.max_reply_eager_short` was not adjusted accordingly. We discussed this with Mikhail and agreed that the common adjustment must be done for this limit as well